en/docs/develop/extending-scim2-user-schemas.md

```diff
@@ -0,0 +1,156 @@
+# Extending SCIM 2.0 User Schemas
+
+The SCIM 2.0 (System for Cross-Domain Identity Management) specification defines a [fixed set of default attributes](http://tools.ietf.org/html/draft-ietf-scim-core-schema-01#section-11.2) for the user object. This set is defined to ensure the interoperability and it can cater to most of the industry's identity management requirements. However, in reality, organizations have their own attributes defined for their users which have already been used in their LDAP schemas. Therefore, SCIM is extensible enough to cope with custom attributes and uses the [Enterprise User Extension](https://tools.ietf.org/html/draft-ietf-scim-core-schema-01#section-11.3) to support extra attributes for the SCIM user object.
+
+WSO2 Identity Server allows users to define their own user schema. These configured schema are then used while creating or validating user objects. This means that custom user attributes can be passed using SCIM for identity management requirements. Follow the steps given below to add a custom attribute. 
+
+## Enable the extension
+
+1. Open the `provisioning-config.xml` file located in the `<IS_HOME>/repository/conf/identity/` folder. 
```

Comment:
configs need to be in .toml format.
However, this config is enabled by default in the 5.9.0.